src/form/v2/birth/advancedSearch.ts

```diff
@@ -20,11 +20,7 @@ export const advancedSearchBirth = [
       description: 'The title of Registration details accordion',
       id: 'v2.advancedSearch.form.registrationDetails'
     },
-    fields: [
-      {
-        fieldId: 'child.dob'
-      }
-    ]
+    fields: []
```

Comment:
Is this empty section left in here on purpose?

Reply:
Yes, currently develop only allows to search inside the declaration data, later, when searching inside metadata will be available, this will be filled up.